examples/subscriptions/app/components/Cart.tsx

```diff
@@ -80,10 +82,12 @@ function CartLineItem({
   line,
 }: {
   layout: CartMainProps['layout'];
-  line: OptimisticCartLine;
+  line: CartLine;
 }) {
   /***********************************************/
   /**********  EXAMPLE UPDATE STARTS  ************/
+  // @ts-expect-error - your project should need this expect error.
+  // It's necessary in our mono repo example setup
   const {id, merchandise, sellingPlanAllocation} = line;
```

Comment:
The instructions for subscriptions tells you to add `sellingPlanAllocation` to the query, yet the example doesn't actually do that. I changed that by adding it to the fragment inside `lib/fragment.ts`, but the diff doesn't seem to properly pick that up when running `npm run typecheck`.